static/app/views/automations/hooks/index.tsx

```diff
@@ -9,8 +10,20 @@ interface UseAutomationsQueryOptions {
 export function useAutomationsQuery(_options: UseAutomationsQueryOptions = {}) {
   const {slug} = useOrganization();
 
-  return useApiQuery<Automation[]>([`/organizations/${slug}/workflows/`], {
+  return useApiQuery<Automation[]>(makeAutomationQueryKey(slug), {
     staleTime: 0,
     retry: false,
   });
 }
+const makeAutomationQueryKey = (orgSlug: string, automationId = ''): [url: string] => [
```

Comment:
there's a type for this
```suggestion
const makeAutomationQueryKey = (orgSlug: string, automationId = ''): ApiQueryKey => [
```